pbs/custom.py

```diff
@@ -41,6 +43,7 @@
 TMPDIR=$(pwd)/tmp runsample.py $SAMPLEDIR {REFPATH} {SAMPLENAME} -od {SAMPLENAME}
 '''
 
+readline.parse_and_bind("tab: complete")
```

Comment:
When I tried this it would not expand like you expect for paths like ../
It would just keep returning the current directories contents